examples/Raytracer_Benchmark/RaytraceBenchmark.cu

```diff
@@ -59,7 +68,8 @@ __global__ void RenderLine(RaytracerData_t rtdata, int py, unsigned char *line)
   if (px >= rtdata.fSize_px) return;
 
   Ray_t ray;
-  adept::Color_t pixel_color = Raytracer::RaytraceOne(rtdata, ray, px, py);
+  adept::BlockData<Ray_t> *rays;
+  adept::Color_t pixel_color = Raytracer::RaytraceOne(rtdata, rays, px, py, 0);
```

Comment:
Can we just drop the other render kernels that are not used by default? (`RenderLine`, `RenderInterlaced`, `RenderTile`)